File: solutions/problem17.js
function checkPrime() {
    let input = prompt("Enter a number to check if it is prime:");
    let num = Number(input);

    if (isNaN(num) || num <= 1) {
        alert("Enter a valid integer greater than 1!");
        return;
    }

    let isPrime = true;

    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) {
            isPrime = false;
            break;
        }
    }

    console.log("Checked Number: " + num);
    
    if (isPrime) {
        console.log("This number is a PRIME number!");
        
    } else {
        console.log("This number is not a prime number!");
       
    }
}

File: solutions/problem-17.py
num = int (input("Enter any positive integer number: "))

if num < 0:
    print("Sorry, not positive integer number !! Try again!")
elif num ==0:
    print("The Factorial of Zero = 1")
else:
    factorial = 1
    i= num
    while i >1:
        factorial*= i
        i -= 1

    print("The Factorial of the number : ",num ," is ", factorial )
